### PR TITLE
Adding readme, maintainers, code of conduct

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Starting with the basics, let's get the language right for our README, maintainer list, and code of conduct.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The current Maintainers Group for the [TODO: Projectname] Project consists of:
+
+| Name | Employer | Responsibilities |
+| ---- | -------- | ---------------- |
+|      |          |                  |
+
```
Comment (a-hilaly) [inline MAINTAINERS.md:6]: Amine Hilaly / AWS / Maintainer

**Comment by a-hilaly** [comment]
Thank you Bridget for putting this together!

File: MAINTAINERS.md (hunk 1 above)
Comment (barney-s) [reply MAINTAINERS.md:6]: Bharanidharan S/ Google/Maintainer

Comment (cheftako) [reply MAINTAINERS.md:6]: Walter F/ Google/Maintainer

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+<!-- TODO: Decide who will handle Code of Conduct reports and replace [INSERT EMAIL ADDRESS]
```
Comment (justinsb) [inline CODE_OF_CONDUCT.md:5]: I think we can merge with the TODO and then work through those as PRs

File: MAINTAINERS.md (hunk 1 above)
Comment (justinsb) [reply MAINTAINERS.md:6]: I think we agreed yesterday to add the "big list" i.e. the 2 or 3 people from each cloud, and then they will self-direct into OWNERs or leads as appropriate.

**Comment by justinsb** [comment]
So I think I can merge this and then we can PR in the verbally agreed list of maintainers and separate PRs to replace the placeholders.  That probably lets us make the most of GitHub to collaborate.

**Comment by justinsb** [comment]
OK, merging and let's follow up with PRs!

**Pull request merged**